Dockerfile

```
@@ -51,14 +51,15 @@ COPY example-config.alloy /etc/alloy/config.alloy
 #
 # NOTE(rfratto): non-root support in Docker containers is an experimental,
 # undocumented feature; use at your own risk.
-RUN groupadd --gid $UID $USERNAME
-RUN useradd -m -u $UID -g $UID $USERNAME
-RUN chown -R $USERNAME:$USERNAME /etc/alloy
-RUN chown -R $USERNAME:$USERNAME /bin/alloy
+RUN groupadd --gid $UID $USERNAME \
+  && useradd -m -u $UID -g $UID $USERNAME \
+  && chown -R $USERNAME:$USERNAME /etc/alloy \
+  && chown -R $USERNAME:$USERNAME /bin/alloy \
+
+  && mkdir -p /var/lib/alloy/data \
+  && chown -R $USERNAME:$USERNAME /var/lib/alloy \
+  && chmod -R 770 /var/lib/alloy
 
-RUN mkdir -p /var/lib/alloy/data
-RUN chown -R $USERNAME:$USERNAME /var/lib/alloy
-RUN chmod -R 770 /var/lib/alloy
 
```

I'd remove the line break and indent 2 more spaces, looks a bit cleaner IMO.
```suggestion
RUN groupadd --gid $UID $USERNAME \
    && useradd -m -u $UID -g $UID $USERNAME \
    && chown -R $USERNAME:$USERNAME /etc/alloy \
    && chown -R $USERNAME:$USERNAME /bin/alloy \
    && mkdir -p /var/lib/alloy/data \
    && chown -R $USERNAME:$USERNAME /var/lib/alloy \
    && chmod -R 770 /var/lib/alloy
```